src/platform/web.rs

```diff
@@ -206,6 +213,21 @@ pub trait EventLoopExtWeb {
     ///
     /// [`ControlFlow::WaitUntil`]: crate::event_loop::ControlFlow::WaitUntil
     fn wait_until_strategy(&self) -> WaitUntilStrategy;
+
+    /// Returns if the users device has multiple screens.
+    ///
+    /// Browsers might always return [`false`] to reduce fingerprinting.
+    fn has_multiple_screens(&self) -> Result<bool, NotSupportedError>;
```

Comment:
Do we really need this call if we have a call to list outputs? Though, maybe web is special.

Reply:
Yes, this is useful to determine if users should even ask for permission.
Asking for permission is an involved process that requires user input through the browser, so its best avoided if not necessary.